src/lib/rest/rest.cpp

```diff
@@ -111,7 +111,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
 
   if (val == NULL || *val == 0)
   {
-    std::string  errorString = std::string("Empty right-hand-side for URI param /") + ckey + "/";
+    std::string  errorString = std::string("Empty right-hand-side for URI param /") + jsonInvalidCharsTransformation(ckey) + "/";
```

Comment:
As I mention in https://github.com/telefonicaid/fiware-orion/issues/2955#issuecomment-335530732, I think is better to fix this at OrionError::toJson(). I guess it should be easy to move the fix.